definitions/divorce/json/CaseEvent/CaseEvent-general-referral-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferral",
+    "Name": "General referral",
+    "Description": "General referral",
+    "DisplayOrder": 16,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/general-referral",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "SecurityClassification": "Public"
```

Comment:
are you sure - no check your answer page?

Reply:
Yes no CYA, at least as per requirements now

Reply:
ok, cool